### DISCUSSION TITLE
Postgres:  UUID as primary key

### DISCUSSION THREAD

**Discussion by tricaricom** [serverpod/serverpod#2641] [Q&A]
Is there anything on the roadmap to use UUIDs instead of integer-based primary keys?

**Comment by muhammedrashidm** [comment]
> Is there anything on the roadmap to use UUIDs instead of integer-based primary keys?

Any updates on this?

**Comment by macMikey** [comment]
dumb question: why not use integer id's for pk's, make them private, and then add uuid's as a separate field?
that's what we generally do with some other db tools.

**Comment by christerswahn** [comment]
This has been solved as of #3659!
